app/src/main/java/org/fossasia/openevent/general/event/EventsRecyclerAdapter.kt

```diff
@@ -34,6 +34,10 @@ class EventsRecyclerAdapter : RecyclerView.Adapter<EventViewHolder>() {
         this.events.addAll(eventList)
     }
 
+    fun removeAll(clear: Boolean) {
+        if (clear) events.clear()
```

Comment:
What is the use of this boolean? If it is false, you won't removeAll the elements?

Reply:
Yes.

Reply:
I think I should check that before calling. Doing that.